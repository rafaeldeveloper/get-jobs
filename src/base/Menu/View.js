import React from 'react'
import { Menu, Icon } from 'semantic-ui-react'

export default ({ action, logo }) => (
    <Menu stackable>
        <Menu.Item>
            <img src={logo} alt="logo" />
        </Menu.Item>
        <Menu.Item
            onClick={() => window.location.replace("list")}
            name='vacancies'
        >
            Vacancies
        </Menu.Item>

        {
            action &&
            <Menu.Item
                position='right'
                name='add'
                onClick={() => action.execute(true)}
            >
                <Icon name={action.icon} />
                {action.name}
            </Menu.Item>
        }
    </Menu>
)
