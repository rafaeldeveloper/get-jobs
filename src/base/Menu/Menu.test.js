import React from 'react';
import Menu from 'base/Menu';
import renderer from 'react-test-renderer';




test('Menu renders correctly', () => {

    const component = renderer.create(
        <Menu/>
    );

    let tree = component.toJSON();
    
    expect(tree).toMatchSnapshot();
''
});

