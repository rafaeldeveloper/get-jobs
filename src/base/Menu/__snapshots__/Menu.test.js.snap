// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<div
  className="ui stackable menu"
>
  <div
    className="item"
    onClick={[Function]}
  >
    <img
      alt="logo"
    />
  </div>
  <a
    className="item"
    onClick={[Function]}
  >
    Vacancies
  </a>
</div>
`;
